File: Nearest Exit.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int s;
        scanf("%d",&s);
        if(s<=50)
        printf("LEFT\n");
        else
        printf("RIGHT\n");
    }
}

File: Nearest Exit.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int n=sc.nextInt();
            if(n>50)
            System.out.println("RIGHT");
            else
            System.out.println("LEFT");
        }
    }
}